diff --git a/C:/Users/sayan/AppData/Local/Temp/dm/package/dist/chunk-DK4WIVY6.mjs b/node_modules/@medusajs/dashboard/dist/chunk-DK4WIVY6.mjs
index f4e1115..dbf3ce9 100644
--- a/C:/Users/sayan/AppData/Local/Temp/dm/package/dist/chunk-DK4WIVY6.mjs
+++ b/node_modules/@medusajs/dashboard/dist/chunk-DK4WIVY6.mjs
@@ -291,7 +291,7 @@ var currencies = {
   INR: {
     code: "INR",
     name: "Indian Rupee",
-    symbol_native: "\u20B9",
+    symbol_native: "paisa",
     decimal_digits: 2
   },
   IQD: {
